compio-io/src/read/buf.rs

```diff
@@ -0,0 +1,33 @@
+use std::io::Result as IoResult;
+
+
```

Comment:
Nit: why extra empty line here?

Reply:
Cuz it's not completed and I only checked-in the completed part.